404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
   <li>Worldwide payments</li>
-  <li>Low or zero processing fees</li>
```

Comment:
The motivation for this PR is people on the -dev mailing list saying that we have been misleading people into thinking that the current state of zero-fee and low-fee transactions will continue in perpetuity.

I think that's what this statement does: it makes people think that low fees are a fundamental part of the system, not just the current state.  (This same feedback also applies to two of your other diff comments: [1](https://github.com/bitcoin-dot-org/bitcoin.org/pull/972#discussion_r35388090) and [2](https://github.com/bitcoin-dot-org/bitcoin.org/pull/972#discussion_r35388128))